ext/node/ops/fs.rs

```diff
@@ -185,16 +185,24 @@ where
     let mut bytes_per_sector = 0;
     let mut available_clusters = 0;
     let mut total_clusters = 0;
-    // SAFETY: Normal GetDiskFreeSpaceW usage.
-    let code = unsafe {
-      GetDiskFreeSpaceW(
-        root.as_ptr(),
-        &mut sectors_per_cluster,
-        &mut bytes_per_sector,
-        &mut available_clusters,
-        &mut total_clusters,
-      )
-    };
+    let mut code = 0;
+    let mut retries = 0;
+    while code == 0 && retries < 10 {
+      if retries > 0 {
+        std::thread::sleep(std::time::Duration::from_millis(1));
```

Comment:
I believe this will actually sleep for ~15ms on windows. Maybe we should just copy exactly what libuv does?